go-packages/meep-model/model.go

```diff
@@ -37,6 +37,7 @@ const activeScenarioKey = "ctrl-engine:active"
 
 // Event types (basic)
 const (
+	EventInit      = "INIT"
```

Comment:
Question - what is INIT event used for?

Reply:
The model Event types were removed when the message queue communications package was integrated.